src/library/simplified/extension/marketing-ab-tests.js

```diff
@@ -0,0 +1,16 @@
+utag_data['ABTestsExpValues'] = '';
+var tpid = utag_data['context.site.tpid'];
+var eapid = utag_data['context.site.eapid'];
+var guid = utag_data['guid'];
+var domain_name = utag_data['context.site.siteName'];
+var expId = utag_data['marketingABTests'];
```

Comment:
We are allowing partners to append their expIds in comma separated format, so that we can evaluate it in this extension.
I have attached the screenshot  by considering multiple marketingABTests. 
Also updated the confluence
 https://confluence/pages/viewpage.action?pageId=805487004